README.md

```diff
@@ -142,6 +142,19 @@ You can address the error `ssh: handshake failed: knownhosts: key is unknown ` w
 - Calling `ssh-keyscan -H github.com >> ~/.ssh/known_hosts` prior to pushing your vergo tag to introduce github to your known hosts.
 - Calling `vergo` with the `--disable-strict-host-check` flag. This should only be used on CI where known hosts are not cached.
 
+## Using GITHUB_TOKEN inside GitHub Actions
+
+Vergo with prefer to use Token Bearer Authentication when running inside a GitHub Action/Workflow.
```

Comment:
What does "with prefer" mean ... is it a typo?

Reply:
Ive fixed this! Sorry about that. Added more detail there